src/index.html

```diff
@@ -4,9 +4,458 @@
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img src="./images/logo.png" alt="header__logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="catalog">
+      <article class="card catalog__card" data-qa="card">
+        <img
+          src=".//images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          class="card__img"
+        >
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="stars card__stars stars--4">
+          <ul class="stars__list">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <p class="stars__desc">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <h2 class="card__price-title">
+            Price:
+          </h2>
+
+          <p class="card__price-number">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__btn"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article class="card catalog__card">
+        <img
+          src=".//images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          class="card__img"
+        >
+
+        <h1 class="card__title">
```

Comment:
Your website should have only one h1 tag. If you have more than one h1 tag on a page, change the other h1 tags to an h2 or h3.